src/loading.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+require('material-design-lite/material.min.css');
+
+const React = require('react');
+const mdl = require('material-design-lite/material.min');
+const classnames = require('classnames');
+
+class Loading extends React.Component {
+
+  constructor(props) {
+    super(props);
```

Comment:
I prefer to use `...args` in the function signature and `super` calls because React may (now or in the future) pass more than just props